fiduswriter/base/static/js/modules/common/dialog.js

```diff
@@ -388,13 +388,18 @@ export class Dialog {
         })
     }
 
-    getHighestDialogZIndex() {
+    nextDialogZIndex() {
         let zIndex = 100
         document
             .querySelectorAll("div.ui-dialog")
             .forEach(
                 dialogEl => (zIndex = Math.max(zIndex, dialogEl.style.zIndex))
             )
+        zIndex += 2
+        document.documentElement.style.setProperty(
```

Comment:
@neuged Have you checked what happens when you navigate between different documents? If I recall correctly, when we navigate between pages, we exchange the `document.body`, while the `document` stays the same. So anything you connect to the document instance will stay the same between page navigations. Not sure if that matters here.